pkg/queue/manager_test.go

```diff
@@ -710,6 +710,10 @@ func TestHeadsAsync(t *testing.T) {
 	now := time.Now().Truncate(time.Second)
 	cq := utiltesting.MakeClusterQueue("fooCq").Obj()
 	wl := kueue.Workload{
+		TypeMeta: metav1.TypeMeta{
```

Comment:
why is this necessary?

Reply:
We get workloads from api server now which contains these necessary fields.

Reply:
Maybe use `cmpopts.IgnoreTypes` instead.